src/apps/deskflow-gui/widgets/ScreenSetupView.cpp

```diff
@@ -140,8 +148,16 @@ void ScreenSetupView::startDrag(Qt::DropActions)
 
 void ScreenSetupView::initViewItemOption(QStyleOptionViewItem *option) const
 {
+  // HACK make a basic widget and init from it
+  auto w = new QWidget();
+  option->initFrom(w);
+  w->deleteLater();
+  delete w;
+
+  option->decorationSize = QSize(96, 96);
   option->showDecorationSelected = true;
   option->decorationPosition = QStyleOptionViewItem::Top;
-  option->displayAlignment = Qt::AlignCenter;
+  option->decorationAlignment = Qt::AlignHCenter | Qt::AlignVCenter;
+  option->displayAlignment = Qt::AlignBottom | Qt::AlignHCenter;
```

Comment:
Align top looks neater.
```suggestion
  option->displayAlignment = Qt::AlignTop | Qt::AlignHCenter;
```

Before:
![image](https://github.com/user-attachments/assets/e3dda01a-8262-4447-9530-1e5244cf2acf)

After:
![Screenshot From 2025-01-30 12-42-10](https://github.com/user-attachments/assets/e8454f07-003d-42fa-8d40-c2b2a81e0218)